providers/oidc.go

```diff
@@ -250,6 +249,10 @@ func (p *OIDCProvider) findClaimsFromIDToken(ctx context.Context, idToken *oidc.
 	// userID claim was not present or was empty in the ID Token
 	if claims.UserID == "" {
 		if profileURL == "" {
+			if bearer {
```

Comment:
I would think in Bearer auth mode, a user wouldn't want a profileURL check for every request if one exists and the email claim is empty? For that case, sliding this condition up above the previous line makes more sense.

Leaving it here preserves the most existing functionality though.

Reply:
Changed this in the latest commit -- realized in Bearer auth mode the original code had another bug: In Bearer auth mode `token` was `nil` -- so even if a `profileURL` existed, trying to hit it for missing claims would be pointless since we would be sending an empty access token and not get a response.